pipelines/linkedcomponents-ui-review.yml

```diff
@@ -44,12 +44,18 @@ resources:
 extends:
   # Filename in Azure DevOps Repository
   template: azure-pipelines-linkedcomponents-ui-review.yml@linkedevents-pipelines
-  # parameters:
+  parameters:
     # Application build arguments and config map values as key value pairs.
     # Does not contain all buildArguments or configMap values, the rest located in linkedevents-pipelines
     # Default value to nodeVersion is set on template.
     # The values here will override the values defined in the linkedevents-pipelines repository
-    # buildArgs:
+    buildArgs:
```

Comment:
yep. I'll revert these when this PR is otherwise ready to merge. Tests in pipeline fails if these are not set

My plan is to add these to [linkedcomponents-ui-common.yml](https://dev.azure.com/City-of-Helsinki/linkedevents/_git/linkedevents-pipelines?path=/vars/linkedcomponents-ui-common.yml&version=GBmaster)
```
REACT_APP_MATOMO_URL_BASE="//matomo.hel.fi/"
REACT_APP_MATOMO_SITE_ID=42
REACT_APP_MATOMO_SRC_URL="matomo.js"
REACT_APP_MATOMO_ENABLED=false
```

And for [linkedcomponents-ui-production.yml](https://dev.azure.com/City-of-Helsinki/linkedevents/_git/linkedevents-pipelines?path=/vars/linkedcomponents-ui-production.yml&version=GBmaster) to override common behaviour
```
REACT_APP_MATOMO_ENABLED=true
```

also old Matomo params need to be removed. And probably can't be removed from production before the new code is there.